nifi-extension-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/AbstractExecuteSQL.java

```diff
@@ -188,6 +189,17 @@ public abstract class AbstractExecuteSQL extends AbstractProcessor {
             .required(true)
             .build();
 
+    public static final PropertyDescriptor OVERWRITE_FLOW_FILE_CONTENT = new PropertyDescriptor.Builder()
+            .name("Overwrite FlowFile Content")
```

Comment:
Rather than introducing a Boolean property, it seems like it would be better to call this something like `Content Output Strategy`, with values such as `Results` and `Ignored`. That would provide more potential for future flexibility.